apps/api/src/service/applicationActionService.ts

```diff
@@ -62,43 +87,43 @@ const applicationActionSvc = (db: PostgresDb) => {
 	};
 
 	return {
-		create: async (application: ApplicationModel) =>
+		create: async (application: ApplicationRecord) =>
```

Comment:
As discussed offline, we don't need to have a `create` action recorded. All the information about when/who created the application is stored in the application itself, having this state transition captures nothing new.

All other actions here are good and should stay! :)

Reply:
Create removed, requires migration to take out the CREATE action: [Remove 'create' actions](https://github.com/Pan-Canadian-Genome-Library/daco/pull/153/commits/833ab877ba6c7ff6e85173c05879d118e65e7200)